docs/documentation/securing_apps/topics/oidc/recommendations.adoc

```diff
@@ -0,0 +1,39 @@
+=== Recommendations
+
+This section describes some recommendations when security your applications with {project_name}.
+
+==== Validating access tokens
+
+If you need to manually validate access tokens issued by {project_name} you can invoke the <<_token_introspection_endpoint,Introspection Endpoint>>.
+The downside to this approach is that you have to make a network invocation to the {project_name} server.  This can be slow and possibly overload the
+server if you have too many validation requests going on at the same time.  {project_name} issued access tokens are https://datatracker.ietf.org/doc/html/rfc7519[JSON Web Tokens (JWT)] digitally signed and encoded using https://datatracker.ietf.org/doc/html/rfc7515[JSON Web Signature (JWS)].
+Because they are encoded in this way, this allows you to locally validate access tokens using the public key of the issuing realm.  You can either hard code the
```

Comment:
```suggestion
Because they are encoded in this way, you can locally validate access tokens using the public key of the issuing realm.  You can either hard code the
```